src/main/java/org/kawanfw/sql/api/server/blob/DefaultBlobUploadConfigurator.java

```diff
@@ -21,20 +34,6 @@
 import java.nio.file.StandardCopyOption;
 import java.util.Date;
 
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-import org.apache.commons.io.FileUtils;
-import org.apache.commons.lang3.SystemUtils;
-import org.apache.tomcat.util.http.fileupload.FileItemIterator;
-import org.apache.tomcat.util.http.fileupload.FileItemStream;
-import org.apache.tomcat.util.http.fileupload.FileUploadException;
-import org.apache.tomcat.util.http.fileupload.disk.DiskFileItemFactory;
-import org.apache.tomcat.util.http.fileupload.servlet.ServletFileUpload;

```

Comment:
Primary item that no longer exists.

Looks like you have to pick a javax OR jakarta concrete now.